android/src/main/java/com/reactnativekalturaplayer/KalturaPlayerRN.kt

```diff
@@ -654,6 +647,7 @@ class KalturaPlayerRN(
                             if (mediaAsset.initialVolume >= 0 && mediaAsset.initialVolume < 1.0) {
                                 player?.setVolume(mediaAsset.initialVolume)
                             }
+                            player?.startPosition = mediaAsset.startPosition
```

Comment:
Check the DVR live media with Start Position.

Reply:
Fixed this issue. If StartPosition is given then only consider it else don't pass it to the player.